features/frameworks/COMPONENT_FPGA_CI_TEST_SHIELD/source/MbedTester.cpp

```diff
@@ -26,23 +26,24 @@
 
 #define mbed_tester_printf(...)
 
-#define PHYSICAL_PINS           128
-#define LOGICAL_PINS            8
-#define FIRMWARE_SIZE           2192012
-#define FIRMWARE_REGION_SIZE    0x220000
-#define FIRMWARE_HEADER_SIZE    0x10000
-#define FLASH_SECTOR_SIZE       0x1000
-#define LENGTH_SIZE             0x4
-#define CRC_SIZE                0x4
-#define FLASH_SPI_FREQ_HZ       2000000
-#define ANALOG_COUNT            4
-
-#define PHYSICAL_NC    ((MbedTester::PhysicalIndex)0xFF)
-
-static const uint8_t KEY[8] = {
-    0x92, 0x9d, 0x9a, 0x9b,
-    0x29, 0x35, 0xa2, 0x65
-};
+namespace {
+    const auto physical_pins = 128;
```

Comment:
`auto` in the limited context of a function is fine, but here in the global space how can you guess what type that const will take ?

Reply:
Done